install/mobile/android/install.yml

```diff
@@ -0,0 +1,14 @@
+id: android-docs
```

Comment:
I'm curious why this ID is using the term "docs" in it? The intention would be to use this install plan for other installation cases, besides just viewing docs links, so I'd suggest to name this more declarative  like `android-mobile`

Reply:
I simply copied this from the browser quickstart, which used this -docs convention. Will change per your suggestion